stories/Molecules/Blocks/CountryCard/countrycard.scss

```diff
@@ -43,7 +39,7 @@
       .country {
         &-card {
           &__caption {
-            transform: translateY(-100%);
+            transform: translateY(calc(-100% + 2.75rem));
```

Comment:
@Ashish-srij On what condition is the value `2.75rem` based on? Can we use a global variable here?

Reply:
@shishirphadke we need to content move on hover and stay at bottom dynmicly. so we r using the calc here. and we don't have a variable for 2.75rem. I have tried to make a variable but variable is not working inside the calc().

Reply:
@Ashish-srij We can use it like this:
```
transform: translateY(calc(-100% + #{$spacing-44}));
```